testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/AbstractTestHiveViews.java

```diff
@@ -310,25 +310,25 @@ public void testTimestampHiveView()
 
         assertThat(query("SELECT CAST(ts AS varchar) FROM timestamp_hive_view")).containsOnly(row("1990-01-02 12:13:14.123"));
         assertThatThrownBy(
-                // TODO(https://github.com/prestosql/presto/issues/6295) it is not possible to query Hive view with timestamps if hive.timestamp-precision=NANOSECONDS
+                // TODO(https://github.com/prestosql/presto/issues/6295) column types from stored view definition should be altered
+                // based on read-time timestamp_precision value
```

Comment:
I'd prefer if this line was either joined to the previous line or had a few spaces before the first word (so IntelliJ counts it as part of the same TODO item).